README.md

```diff
@@ -91,6 +91,102 @@ Provisioner](https://www.packer.io/docs/provisioners/shell.html) runs the
 necessary configuration tasks.  Then, Packer creates an AMI from the instance
 and terminates the instance after the AMI is created.
 
+### Container Image Caching
+
+By default, some container images are cached during the AMI build process in order to reduce the latency of the node getting to a `Ready` state when launched. 
```

Comment:
Should it be changed after `cache_container_images` default value changed from `true` to `false`?

Commit: https://github.com/awslabs/amazon-eks-ami/pull/938/commits/9158b7a47351f08ecb794e52823c46cd9428d5cd